src/main/java/com/galvanize/alpha/speedwaytrials/drivers/controller/DriversController.java

```diff
@@ -13,17 +15,21 @@
 
 @RestController
 @RequestMapping("drivers")
+@AllArgsConstructor
```

Comment:
This is odd in a controller. Did you mean to create a default constructor with the `DriverService` being passed in with **Depenedency Injection**?

Reply:
@davidgrieser  Would you suggest we refactor to the boilerplate constructor syntax?  We were thinking that the AllArgsConstructor would force the DriverService dependency to be injected into the Controller, and that a future refactor was always possible if we needed more fine-grained handling of the controller instantiation.